templates/trust/package.json

```diff
@@ -43,6 +43,7 @@
     "@aragon/apps-agent": "^1.0.0",
     "@aragon/apps-vault": "^4.1.0",
     "@aragon/apps-voting": "^2.1.0",
+    "@aragon/apps-payroll": "^1.0.0-rc.1",
```

Comment:
Don't think we need this dependency here 😄

Reply:
There's something weird going on, if that dependency isn't there, Trust doesn't compile. Or at least that was the case with the CI.

Reply:
Hmm, I see, the contract resolution algorithm is a bit tricky with how npm works (and it's why we can't hoist contracts when using lerna).

Not sure if truffle@5 has made any inroads to fixing this, but for now I've moved the unused apps into the dev dependencies.